test/Microsoft.AspNet.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.Image.html

```diff
@@ -9,7 +9,7 @@
 
     <h2>Image Tag Helper Test</h2>
     <!-- Plain image tag -->
-    <img src="/images/red.png" alt="Red block" title="&lt;the title>">
+    <img src="/images/red.png" alt="Red block" title="&lt;the title>" />
```

Comment:
The `ImageTagHelper` only applied to elements with the src and append-file-version attribute combo